README.md

```diff
@@ -12,15 +12,22 @@ A Quick-start Guide for people that want to **D**o **W**hat **Y**ou **L**ove!
 The world is not perfect.  
 We have all worked somewhere we _didn't **love**_. <br />
 We have _all_ felt the _frustration_ of using products that have flaws
-(and been unable to fix them).   
-**The aim of _dwyl_ is to address *both* of these issues _simultaneously_.**
-
-If you'd like to know more, _please **read our [manifesto](/manifesto.md)**_.  
+that _we_ were unable to fix ourselves.   
+We have all experienced being in an environment that
+_didn't support us_ in doing all we can do in the world.
+**The aim of _dwyl_ is to address *all* of these issues _simultaneously_:**
+please read our [**mission**](https://github.com/dwyl/start-here/blob/master/mission.md)
```

Comment:
```suggestion
please read our [**mission**](https://github.com/dwyl/start-here/blob/master/mission.md#what-is-dwyls-mission)
```
How about _deep_ linking directly to the _answer_ so people don't have to scroll/read lots of "background" ...?

Reply:
Included in a _separate_ commit by @iteles https://github.com/dwyl/start-here/pull/197#issuecomment-578840771 ⬇️